doc/upgrading.md

```diff
@@ -13,6 +14,30 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 30.0
+
+* [RMB-246](https://issues.folio.org/browse/RMB-246) Switch to vertx-pg-client.
+  * Class `SQLConnection` is now provided by RMB. But it was part of
+    old SQL client. `io.vertx.ext.sql.SQLConnection` ->
+    `org.folio.rest.persist.SQLConnection`.
+  * All functions that previusly returned `UpdateResult` now returns
+    `RowSet<Row>`. From that result the number of rows affacted by
+    SQL was `getUpdated()` it is now `rowCount()`.
+  * All functions that previusly returned `ResultSet` now returns
+    `RowSet<Row>`. From that result, the number of rows affacted by
+    SQL is `rowCount()`. The size() method returns number of rows
+    returned. An iterator to go through rows is obtained by calling
+    `iterator`.
+  * `PostgresClient.getClient()` is no longer public. If you need a
+    connection, use PostgresClient.startTx(). If you don't have
+    transactions, you don't need access to SQLConnection.
+  * Exceptions thrown by new client is `io.vertx.pgclient.PgException`.
+    Was com.github.jasync.sql.db.postgresql.exceptions.GenericDatabaseException
+    before. The getMessage() only contains message! Not details, code.
+  * `PgExceptionFacade.getTable` removed.
+  * `PgExceptionFacade.getIndex` removed.
+  * `PostgresClient.mutate` removed (deprecated since Oct 2018).
```

Comment:
There are several PostgresClient.get methods that have been deprecated since 6 months. Should we remove them as well?

Reply:
I think it depends on how much they are used.

Reply:
I went through all usages of deprecated methods reported by sonar and could not find any deprecated PostgresClient.get usage:
https://sonarcloud.io/organizations/folio-org/rules?open=java%3AS1874&q=depreca#coding-rules-detail-quality-profiles

Reply:
Therefore I suggest to remove them.